pil_draw_text_block.py

```diff
@@ -0,0 +1,43 @@
+from PIL import Image
+from PIL import ImageDraw
+from PIL import ImageFont
+from nio.common.block.base import Block
+from nio.common.discovery import Discoverable, DiscoverableType
+from nio.metadata.properties import ExpressionProperty, VersionProperty, \
+        ObjectProperty, PropertyHolder
+from nio.util.environment import NIOEnvironment
+
+
+class Coordinate(PropertyHolder):
+    x = ExpressionProperty(default='0')
+    y = ExpressionProperty(default='0')
+
+
+@Discoverable(DiscoverableType.block)
+class PILDrawText(Block):
+
+    """ Draw text to a PIL Image """
+
+    version = VersionProperty('0.1.0')
+    text = ExpressionProperty(title='Text', default='Hello, n.io!')
+    coordinate = ObjectProperty(Coordinate, default=Coordinate())
+
+    def process_signals(self, signals, input_id='default'):
+        for signal in signals:
+            self._draw_text(signal)
+        self.notify_signals(signals, output_id='default')
+
+    def _draw_text(self, signal):
+        """ Edits the Image object in signal.image by drawing text to it """
+        self._logger.debug('Create Draw object')
+        draw = ImageDraw.Draw(signal.image)
```

Comment:
We should probably make this a hidden configurable expression, rather than assuming the block has `image` on it. Separate issue or want to do it here?